diff --git a/w/extensions/EmbedVideo/EmbedVideo.hooks.php b/w/extensions/EmbedVideo/EmbedVideo.hooks.php
index 0fb64d3..7b53efd 100644
--- a/w/extensions/EmbedVideo/EmbedVideo.hooks.php
+++ b/w/extensions/EmbedVideo/EmbedVideo.hooks.php
@@ -514,16 +514,16 @@ class EmbedVideoHooks {
 	static public function parseEV( $parser, $service = null, $id = null, $dimensions = null, $alignment = null, $description = null, $container = null, $urlArgs = null, $autoResize = null, $vAlignment = null ) {
 		self::resetParameters();
 
-		$service		= trim( $service );
-		$id				= trim( $id );
-		$alignment		= trim( $alignment );
-		$description	= trim( $description );
-		$dimensions		= trim( $dimensions );
-		$urlArgs		= trim( $urlArgs );
+		$service		= is_null($service) ? '' : trim( $service );
+		$id				= is_null($id) ? '' : trim( $id );
+		$alignment		= is_null($alignment) ? '' : trim( $alignment );
+		$description	= is_null($description) ? '' : trim( $description );
+		$dimensions		= is_null($dimensions) ? '' : trim( $dimensions );
+		$urlArgs		= is_null($urlArgs) ? '' : trim( $urlArgs );
 		$width			= null;
 		$height			= null;
-		$autoResize		= ( isset( $autoResize ) && strtolower( trim( $autoResize ) ) == "false" ) ? false : true;
-		$vAlignment		= trim( $vAlignment );
+		$autoResize		= ( isset( $autoResize ) && strtolower( is_null($autoResize) ? '' : trim( $autoResize ) ) == "false" ) ? false : true;
+		$vAlignment		= trim( is_null($vAlignment) ? '' : $vAlignment );
 
 		// I am not using $parser->parseWidthParam() since it can not handle height only.  Example: x100
 		if ( stristr( $dimensions, 'x' ) ) {
