diff --git a/w/extensions/EmbedVideo/EmbedVideo.hooks.php b/w/extensions/EmbedVideo/EmbedVideo.hooks.php
index 0fb64d3..7df3301 100644
--- a/w/extensions/EmbedVideo/EmbedVideo.hooks.php
+++ b/w/extensions/EmbedVideo/EmbedVideo.hooks.php
@@ -70,16 +70,14 @@ class EmbedVideoHooks {
 	 * @return	void
 	 */
 	public static function onExtension() {
-		global $wgEmbedVideoDefaultWidth, $wgMediaHandlers, $wgFileExtensions;
-
-		$config = ConfigFactory::getDefaultInstance()->makeConfig('main');
+		global $wgEmbedVideoDefaultWidth, $wgMediaHandlers, $wgFileExtensions, $wgEmbedVideoEnableAudioHandler, $wgEmbedVideoEnableVideoHandler, $wgEmbedVideoAddFileExtensions;
 
 		if (!isset($wgEmbedVideoDefaultWidth) && (isset($_SERVER['HTTP_X_MOBILE']) && $_SERVER['HTTP_X_MOBILE'] == 'true') && $_COOKIE['stopMobileRedirect'] != 1) {
 			//Set a smaller default width when in mobile view.
 			$wgEmbedVideoDefaultWidth = 320;
 		}
 
-		if ($config->get('EmbedVideoEnableAudioHandler')) {
+		if ($wgEmbedVideoEnableAudioHandler) {
 			$wgMediaHandlers['application/ogg']		= 'EmbedVideo\AudioHandler';
 			$wgMediaHandlers['audio/flac']			= 'EmbedVideo\AudioHandler';
 			$wgMediaHandlers['audio/ogg']			= 'EmbedVideo\AudioHandler';
@@ -89,7 +87,7 @@ class EmbedVideoHooks {
 			$wgMediaHandlers['audio/webm']			= 'EmbedVideo\AudioHandler';
 			$wgMediaHandlers['audio/x-flac']		= 'EmbedVideo\AudioHandler';
 		}
-		if ($config->get('EmbedVideoEnableVideoHandler')) {
+		if ($wgEmbedVideoEnableAudioHandler) {
 			$wgMediaHandlers['video/mp4']			= 'EmbedVideo\VideoHandler';
 			$wgMediaHandlers['video/ogg']			= 'EmbedVideo\VideoHandler';
 			$wgMediaHandlers['video/quicktime']		= 'EmbedVideo\VideoHandler';
@@ -97,7 +95,7 @@ class EmbedVideoHooks {
 			$wgMediaHandlers['video/x-matroska']	= 'EmbedVideo\VideoHandler';
 		}
 
-		if ($config->get('EmbedVideoAddFileExtensions')) {
+		if ($wgEmbedVideoAddFileExtensions) {
 			$wgFileExtensions[] = 'flac';
 			$wgFileExtensions[] = 'mkv';
 			$wgFileExtensions[] = 'mov';
