diff --git a/w/extensions/EmbedVideo/EmbedVideo.hooks.php b/w/extensions/EmbedVideo/EmbedVideo.hooks.php
index 0fb64d3..9860d52 100644
--- a/w/extensions/EmbedVideo/EmbedVideo.hooks.php
+++ b/w/extensions/EmbedVideo/EmbedVideo.hooks.php
@@ -318,7 +318,7 @@ class EmbedVideoHooks {
 				$parser,
 				$args['service'],
 				$args['defaultid'],
-				"${w}x${h}",
+				"{$w}x{$h}",
 				$args['alignment'],
 				$args['description'],
 				$args['container'],
